### DISCUSSION TITLE
Leaving GotR pre-maturely

### DISCUSSION THREAD

**Discussion by averylipsit** [runelite/runelite#16913] [Ideas]
Someone should make a checkbox or something to make it more difficult to leave GotR too early. I'm not sure if other people are as incompetent as I am but I've done this 3 times now and I would think that this wouldn't be super complicated to create (although I have no idea) and helpful for those of us who suck at paying attention. XD

**Comment by Adam-** [comment]
Turn off the quick-pass swap in the menu entry swapper.

**Reply by averylipsit** [reply]
Brilliant!!! Thanks a bunch :)
